core/App/components/inputs/CheckBoxRow.tsx

```diff
@@ -38,9 +38,9 @@ const CheckBoxRow: React.FC<Props> = ({ title, accessibilityLabel, testID, check
         onPress={onPress}
       >
         {checked ? (
-          <Icon name={'check-box'} size={36} color={ColorPallet.brand.primary} />
+          <Icon name={'check-box'} size={36} color={Inputs.checkBoxColor.color} />
```

Comment:
Minor nit: Do we want to mix case here? I'm ok with just using camelCase altogether

Reply:
+1 For not mixing cases. Let's go with what the app primarily uses now.

Reply:
The existing style objects are using pascal case, did you want me to update those as well? (Buttons, ColorPallet, etc)

Reply:
So for instance with Buttons.primary, do we want the B to remain capitalized?